localstack/aws/handlers/response_aggregator.py

```diff
@@ -0,0 +1,18 @@
+from localstack import config
+from localstack.aws.api import RequestContext
+from localstack.aws.chain import HandlerChain
+from localstack.http import Response
+from localstack.utils.analytics.response_aggregator import ResponseAggregator
+
+
+class ResponseAggregatorHandler:
+    def __init__(self):
+        self.aggregator = ResponseAggregator()
+        self.aggregator.start_thread()
```

Comment:
the install is failing because the initialization of `ResponseAggregatorHandler` happens on module level, and [plux](https://github.com/localstack/plux) imports all the code at install time to resolve plugins. because the thread is not a daemon thread, it stays open and blocks the `setup.py plugins` call.

we should maybe lazily initialize the aggregator the first time it's being invoked. or find some other way of not starting it during module import.